chirp/drivers/th9000.py

```diff
@@ -399,8 +399,13 @@ def _checksum(data):
 def _read(radio, length):
     try:
         data = radio.pipe.read(length)
+<<<<<<< HEAD
```

Comment:
Can you please rebase on py3 and not just keep merging into your PR? Things like this are now permanently in your PR until/unless you clean it up.

I would really prefer a PR full of consistent and coherent changes that fix specific things in a specific way. Each commit should pass tests and build on the last one.